apps/api/src/persistence/persistence.module.ts

```diff
@@ -50,6 +51,13 @@ export class PersistenceModule {
             inject: [ArangoConnectionProvider],
         };
 
+        const arangoQueryRunnerProvider = {
```

Comment:
Do we really need to manually specify the provider for every new export here? Maybe there's a pattern that avoids this?